pkg/commands/compute/init.go

```diff
@@ -1,9 +1,12 @@
 package compute
```

Comment:
Append to...

> Creating a new Compute@Edge project.

...information about the use of `--from` so it's clear that using the from flag has done something (as the 'progress' spinner messaging is maybe too subtle otherwise).